src/backer/cli/commands/start.py

```diff
@@ -87,9 +89,12 @@ def runBacker(name="backer", base="", alias="backer", bran="", tcp=5665, http=56
         hab = hby.makeHab(name=alias, transferable=False)
     if ledger == "cardano":
         ldg = cardaning.Cardano(name=alias, hab=hab, ks=ks)
+
+    ogm = ogmiosing.Ogmioser()
     
-    doers = [hbyDoer]
+    doers = [hbyDoer, ogm]
```

Comment:
Instead of checking if ogmios is on tip within the backer, we could do this:
- Launch ogmioser to sync
- Once we've synced, call `self.extend(backering.setupBacker...` from that file so backer is only launched once we are synced

Reply:
Yep, It's better, I have just do that.